srcpkgs/osmin/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'osmin'
+pkgname=osmin
+version=1.8.7
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ ."
+hostmakedepends="qt5-qmake pkg-config qt5-host-tools git"
+makedepends="qt5-declarative-devel qt5-3d clang libgomp-devel
+	libomp-devel qt5-declarative-devel qt5-tools-devel
```

Comment:
line continuations should be indented with a single space (makedepends, distfiles, checksum)

Reply:
Ok. I did it this way because the linter told me to use tabs instead of spaces... Maybe this check could be added to the linter

Reply:
it's a bit non-trivial to lint, but it should be in xlint in the future (leahneukirchen/xtools#247)